src/app/modules/item/pages/item-progress/item-progress.component.ts

```diff
@@ -21,29 +23,29 @@ export class ItemProgressComponent implements OnChanges {
   @ViewChild('chapterGroupProgressTab') chapterGroupProgressTab?: RouterLinkActive;
   @ViewChild('chapterUserProgressTab') chapterUserProgressTab?: RouterLinkActive;
 
-  readonly session$ = this.sessionService.session$;
+  private itemData$ = new ReplaySubject<ItemData>();
+  selectors$ = combineLatest([ this.itemData$, this.groupWatchingService.watchedGroup$ ]).pipe(
+    map(([ itemData, watchedGroup ]) => {
+      if (!watchedGroup || watchedGroup.route.isUser) {
+        return isATask(itemData.item) ? 'none' : 'withUserProgress';
+      } else {
+        return 'withGroupProgress';
+      }
+    }),
+  );
```

Comment:
Do you have test cases to ensure there's no regression here ?

Reply:
scenarios updated to check this as well